external/dhcpc/dhcpc.c

```diff
@@ -703,15 +719,16 @@ static int dhcpc_request(void *handle, struct dhcpc_state *presult)
 
 #if defined(CONFIG_NETDB_DNSCLIENT) && defined(CONFIG_NETDB_DNSSERVER_BY_DHCP)
 	struct sockaddr_in dns;
```

Comment:
I think dns_add_nameserver isn't used. please remove it if it's not called anymore.

Reply:
dns_add_nameserver is being used at runtime to add the DNS server address.

Reply:
how did you check that dns_add_nameserver is being used on runtime? I can't find definition of it anywhere.

Reply:
by mistake, I checked in some other repo. now I have removed.